### PR TITLE
fix(lualine): change `fg` of section `a` in onedarker

### PR DESCRIPTION
**Pull request by younger-1**
# Description

Move `onedarker` theme from `lua/lvim/lualine/themes/onedarker.lua` to `lua/lualine/themes/onedarker.lua` 
If do not specify `lvim.builtin.lualine.options.theme`, it will defaults to `auto` other than `onedarker` which is set here:

https://github.com/LunarVim/LunarVim/blob/adc0b8b4642508408f6aedb7937ec0fdc2a891d8/lua/lvim/core/lualine/styles.lua#L130-L132

Update:
It is common to set section `a` and `z` the same `fg` color when `bg` is bright color.

https://github.com/nvim-lualine/lualine.nvim/blob/3f5cdc51a08c437c7705e283eebd4cf9fbb18f80/lua/lualine/themes/onedark.lua#L20-L24

## How Has This Been Tested?

Before:
```lua
print(lvim.builtin.lualine.options.theme)   -- 'auto'
```

After:
```lua
print(lvim.builtin.lualine.options.theme)   -- 'onedarker'
```

### CODE REVIEW

**Comment by kylo252** [comment]
I think something wrong happened during your branch merge/update, because the description covers changes that aren't currently present in the PR.

**Comment by younger-1** [comment]
The code above is to show where `lvim.builtin.lualine.options.theme` got value.
I just moved the `onedarker` file to proper place where `lualine` could find the theme and change the `fg` of section `a`
Because the previous change had been merged to rolling branch in [this commit](https://github.com/LunarVim/LunarVim/commit/ebb622539831bddd14776433f0ad8ec939a166ac) by ChristianChiarulli, so I have to resolved conflict by merging rolling branch again.

File: lua/lualine/themes/onedarker.lua [hunk 1]
```diff
@@ -19,7 +19,7 @@ local colors = {
 -- LuaFormatter on
 return {
   normal = {
-    a = { fg = colors.fg, bg = colors.blue, gui = "bold" },
+    a = { fg = colors.bg, bg = colors.blue, gui = "bold" },
```
Comment (kylo252) [inline lua/lualine/themes/onedarker.lua:22]:
```suggestion
    a = { fg = colors.gray2, bg = colors.blue, gui = "bold" },
```

**Review by kylo252 — CHANGES_REQUESTED** [review]
You also need the same changes for other modes
```diff
diff --git a/lua/lualine/themes/onedarker.lua b/lua/lualine/themes/onedarker.lua
index 396657b..f63fd35 100644
--- a/lua/lualine/themes/onedarker.lua
+++ b/lua/lualine/themes/onedarker.lua
@@ -19,13 +19,13 @@ local colors = {
 -- LuaFormatter on
 return {
   normal = {
-    a = { fg = colors.fg, bg = colors.blue, gui = "bold" },
+    a = { fg = colors.gray2, bg = colors.blue, gui = "bold" },
     b = { fg = colors.fg, bg = colors.bg },
     c = { fg = colors.fg, bg = colors.bg },
   },
-  insert = { a = { fg = colors.fg, bg = colors.green, gui = "bold" } },
-  visual = { a = { fg = colors.fg, bg = colors.purple, gui = "bold" } },
-  command = { a = { fg = colors.fg, bg = colors.yellow, gui = "bold" } },
+  insert = { a = { fg = colors.gray2, bg = colors.green, gui = "bold" } },
+  visual = { a = { fg = colors.gray2, bg = colors.purple, gui = "bold" } },
+  command = { a = { fg = colors.gray2, bg = colors.yellow, gui = "bold" } },
   replace = { a = { fg = colors.fg, bg = colors.red1, gui = "bold" } },
   inactive = {
     a = { fg = colors.gray1, bg = colors.bg, gui = "bold" },

```

**Review by kylo252 — APPROVED** [review]
Thank you!

**Pull request merged**